source/_integrations/history_stats.markdown

```diff
@@ -47,7 +47,7 @@ state:
   required: true
   type: string
 name:
-  description: Name displayed on the frontend.
+  description: Name displayed on the frontend. Note that it is used by Home Assistant to generate sensor's `object_id` so it is advisable to choose a unique one and change name for frontend using [customization](https://www.home-assistant.io/docs/configuration/customizing-devices/#friendly_name) or via [Lovelace](https://www.home-assistant.io/lovelace/entities/#name).
```

Comment:
`object_id` ?

Please, remove the domain part from the links and make the links relative to the root of the website.

Reply:
> `object_id` ?

yeah, that's how it's called [here](https://www.home-assistant.io/docs/configuration/state_object/) (as it's definitely not an `entity_id`)

have just converted links to relatives.